modules/tinymce/src/core/test/ts/browser/UndoManagerTest.ts

```diff
@@ -564,4 +565,41 @@ describe('browser.tinymce.core.UndoManager', () => {
     editor.execCommand('mceFocus');
     assert.isFalse(editor.undoManager.hasUndo());
   });
+
+  context('Undo when first element is contenteditable="false"', () => {
+    beforeEach(() => {
+      const editor = hook.editor();
+      editor.undoManager.clear();
+      editor.setDirty(false);
+      editor.setContent('<div contenteditable="false"><p>CEF</p></div><p>something</p><p>something else</p>', { format: 'raw' });
```

Comment:
```suggestion
      editor.resetContent('<div contenteditable="false"><p>CEF</p></div><p>something</p><p>something else</p>');
```
That should be the same? `resetContent` won't allow us to pass as raw, however there's nothing we are including here where raw is required.

Reply:
Looks like this works, nice